microsoft-edge/test-and-automation/test-and-automation.md

```diff
@@ -0,0 +1,86 @@
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: devtools
+ms.date: 02/14/2022
+---
+# Test and automation in Microsoft Edge
+
+There are many tools to automate your testing of Microsoft Edge:
+
+| Tool | Description |
+| --- | --- |
+| DevTools Protocol | Instrument, inspect, debug, and profile browsers including Microsoft Edge. |
+| Origin Trials | Try out experimental APIs on live sites for a limited period of time. |
+| Playwright | The Playwright library provides cross-browser automation through a single API. |
+| Puppeteer | The Puppeteer library provides a high-level API to control Microsoft Edge using the DevTools Protocol. |
+| WebDriver | Automates testing in Microsoft Edge by simulating user interaction.  Provides advantages over JavaScript unit tests. |
+| webhint | Checks your code for best practices and common errors, to test and improve accessibility, performance, cross-browser and PWA compatibility, and security of your site. |
+
+These tools are described below.
+
+
+<!-- ====================================================================== -->
+## DevTools Protocol
+
+Use the DevTools Protocol to instrument, inspect, debug, and profile browsers including Microsoft Edge.  The Microsoft Edge DevTools Protocol matches the APIs of the Chrome DevTools Protocol.
+
+See [DevTools Protocol](devtools-protocol.md).
+
+
+<!-- ====================================================================== -->
+## Origin Trials
+
+You can use Origin Trials to try out experimental APIs on live sites for a limited period of time.  When using Origin Trials, users of Microsoft Edge that visit your site may run code that uses experimental APIs.  To access the experimental APIs on each user machine, you don't need to go to `edge://flags` and turn on feature flags.
+
+See [Use Origin Trials in Microsoft Edge](../origin-trials/index.md).
+
+
+<!-- ====================================================================== -->
+## Playwright
+
+The Playwright library provides cross-browser automation through a single API.  Playwright enables cross-browser web automation that is evergreen, capable, reliable, and fast.
+
+Playwright launches browsers as headless, by default.  Headless browsers don't display a UI; so generally you use the command line - however, you can also configure Playwright to run the full Microsoft Edge UI.
+
+See [Use Playwright to automate and test in Microsoft Edge](../playwright/index.md).
+
+
+<!-- ====================================================================== -->
+## Puppeteer
+
+The Puppeteer library provides a high-level API to control Microsoft Edge using the DevTools Protocol.
```

Comment:
```suggestion
The Puppeteer library provides a high-level API to control Chromium-based browsers like Microsoft Edge using the DevTools Protocol.
```

Reply:
Applied fix 2x:
here, 
top of Puppeteer article.